docs/source/developerguide.rst

```diff
@@ -266,129 +307,157 @@ We can create the streaming inference app by combining the MONAI Stream componen
+                  width=256,
+                  height=256,
+               )
          ),
          # chain output to `NVInferServer`
          NVInferServer(
-            config=infer_server_config,
+               config=infer_server_config,
          ),
          # use `TransformChainComponent` to blend the original image with the segmentation
          # output from `NVInferServer`
          TransformChainComponent(
-            input_labels=["original_image", "seg_output"],
-            output_label="seg_overlay",
-            transform_chain=Compose(
-               [
-                  # apply post-transforms to segmentation
-                  Activationsd(keys=["seg_output"], sigmoid=True),
-                  AsDiscreted(keys=["seg_output"]),
-                  AddChanneld(keys=["seg_output"]),
-                  AsChannelLastd(keys=["seg_output"]),
-                  # merge segmentation and original image into one for viewing
-                  ConcatItemsd(keys=["original_image", "seg_output"], name="seg_overlay", dim=2),
-                  Lambdad(keys=["seg_overlay"], func=color_blender),
-                  CastToTyped(keys="seg_overlay", dtype=np.uint8),
-               ]
-            ),
+               output_label="CONCAT_IMAGE",
+               transform_chain=Compose(
+                  [
+                     # apply post-transforms to segmentation
+                     Activationsd(keys=["OUTPUT__0"], sigmoid=True),
+                     AsDiscreted(keys=["OUTPUT__0"]),
+                     AsChannelLastd(keys=["OUTPUT__0"]),
+                     # concatenate segmentation and original image
+                     CastToTyped(keys=["ORIGINAL_IMAGE"], dtype=np.float),
+                     ConcatItemsd(keys=["ORIGINAL_IMAGE", "OUTPUT__0"], name="CONCAT_IMAGE", dim=2),
+                     # blend the original image and segmentation
+                     Lambdad(keys=["CONCAT_IMAGE"], func=color_blender),
+                     ScaleIntensityd(keys=["CONCAT_IMAGE"], minv=0, maxv=256),
+                     CastToTyped(keys=["CONCAT_IMAGE"], dtype=np.uint8),
+                  ]
+               ),
          ),
          # display output for `TransformChainComponent`
          NVEglGlesSink(sync=True),
-       ]
+      ]
    )
 
    # execute pipeline
    pipeline()
 
+Looking more closely at lines `9` and `10`, the ``NVInferServer`` component expects a model named ``us_unet_256x256``
+under ``/app/models`` with the `directory structure <https://github.com/triton-inference-server/server/blob/main/docs/model_repository.md#repository-layout>`_ 
+and model `configuration <https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md>`_ file
+expected by the `Triton Inference Server <https://github.com/triton-inference-server/server>`_.
+
+In this case the directory structure of the model required the pipeline above is
+
+.. code-block::
+
+   /app/models
+   └── us_unet_256x256
```

Comment:
same as above, this will throw an error that the model directory name does not match (eg, in the US config.pbtxt, it is looking for monai_unet_trt as the folder)

Reply:
config.pbtxt shows
```
name: "us_unet_256x256"
platform: "tensorrt_plan"
default_model_filename: "monai_unet.engine"
```
this is correct afaik?

Reply:
...cool, looks like someone updated the dataset